semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java

```diff
@@ -638,6 +638,34 @@
    */
   public static final AttributeKey<Long> RPC_GRPC_STATUS_CODE = longKey("rpc.grpc.status_code");
 
+  /**
+   * Protocol version as in `jsonrpc` property of request/response. Since JSON-RPC 1.0 does not
+   * specify this, the value can be omitted.
+   */
+  public static final AttributeKey<String> RPC_JSONRPC_VERSION = stringKey("rpc.jsonrpc.version");
+
+  /**
+   * `method` property from request. Unlike `rpc.method`, this may not relate to the actual method
+   * being called. Useful for client-side traces since client does not know what will be called on
+   * the server.
+   */
+  public static final AttributeKey<String> RPC_JSONRPC_METHOD = stringKey("rpc.jsonrpc.method");
```

Comment:
That new attribute is a bit unfortunate, I opened a spec PR: open-telemetry/opentelemetry-specification#1748 to remove it / clarify that rpc.method can be used instead.

Reply:
@Oberon00 do you think we should just wait for that to settle before merging this, then?

Reply:
IMHO it's better to have this one bad attribute than not to update at all.

Reply:
I just cut the 1.3.0 release, so we'll get what we get in the next version. :)